modules/log6x-installing-the-logging-ui-plug-in-cli.adoc

```diff
@@ -0,0 +1,52 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-04-18
+:_mod-docs-content-type: PROCEDURE
+
+[id="installing-the-logging-ui-plug-in-cli_{context}"]
+= Installing the Logging UI plug-in by using the CLI
+
+Install the Logging UI plug-in by using the comman-line interfatce (CLI) to visualize logs.
+ 
+.Prerequisites
+* You have administrator permissions.
+* You installed the {oc-first}.
+* You installed and configured {loki-op}.
+
+.Procedure
+. Install the {coo-full}. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/cluster_observability_operator/installing-cluster-observability-operators[Cluster Observability Operator].
+
+. Create a `UIPlugin` custom resource (CR):
++
+.Example `UIPlugin` CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1alpha1
+kind: UIPlugin
+metadata:
+  name: logging  # <1>
+spec:
+  type: Logging  # <2>
+  logging:
+    lokiStack:
+      name: logging-loki  # <3>
+----
+<1> Set `name` to `logging`.
+<2> Set `type` to `Logging`.
+<3> `name` must match the name of your LokiStack instance. 
++
+[NOTE]
```

Comment:
I think we can remove this line. We don't support lokistack in the other namespace now. That may be enable in feature.

Reply:
I know there was a customer case that required us to add the namespace. But in regards to support I'm not I'll check with @xperimental during the daily today

Reply:
No idea if there is limitation to deploy the logging in the other namespace. let me confirm this.

Reply:
/lgtm